repos/c2sm/packages/flexpart-cosmo/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+#
+from spack import *
+from distutils.dir_util import copy_tree
+from llnl.util.filesystem import working_dir
+
+
+class FlexpartCosmo(MakefilePackage):
+    """flexpart is a Lagrangian dispersion model"""
+
+    homepage = 'https://github.com/C2SM-RCM/flexpart'
+    git = 'ssh://git@github.com/C2SM-RCM/flexpart.git'
+
+    version('main', branch='testing')
```

Comment:
```suggestion
    version('main', branch='main')
```
Maybe a left-over from testing.

Reply:
The `testing` branch is not merged yet. I hope to get this done by the end of this week.